checker/bin-devel/test-misc.sh

```diff
@@ -71,56 +71,7 @@ git diff --exit-code docs/manual/contributors.tex || \
      echo "    and remake contributors.tex after that pull request is merged." &&
      false)
 
-# Check the definition of qualifiers in Checker Framework against the JDK
-echo "Checking the definition of qualifiers in Checker Framework against the JDK"
-CURRENT_PATH=$(pwd)
-src_dir="$CURRENT_PATH/checker-qual/src/main/java/org/checkerframework"
-jdk_dir="$CURRENT_PATH/../jdk/src/java.base/share/classes/org/checkerframework"
 set +o xtrace  # Turn off xtrace to avoid verbose output
```

Comment:
Should the set/unset be part of the script? It would make the script more self-contained.